overlord/snapstate/snapmgr_test.go

```diff
@@ -641,7 +642,7 @@ func (s *snapmgrTestSuite) TestInstallRunThrough(c *C) {
 		},
 		{
 			op:   "copy-data",
-			name: "/snap/some-snap/11",
+			name: fmt.Sprintf("%s/some-snap/11", dirs.SnapMountDir),
```

Comment:
I want the tests to pass if that path is changed. This way we know that there's nothing hardcoded / hidden and buggy when the path is different.